rotkehlchen/chain/ethereum/modules/l2/loopring.py

```diff
@@ -322,7 +322,7 @@ def get_account_balances(self, account_id: int) -> Dict[Asset, Balance]:
         for balance_entry in response:
             try:
                 token_id = balance_entry['tokenId']
-                total = balance_entry['total']
+                total = int(balance_entry['total'])
```

Comment:
No this is not the right way. You are forcing an int conversion here that may fail and raise a ValueError.

You should use the appropriate desierialize function. This one: https://github.com/rotki/rotki/blob/8833d0f327a3039cf338d20af03951738f4a0d49/rotkehlchen/serialization/deserialize.py#L691-L702

Then any `DeserializationError` that may be raised would be handled below.